src/satbias/README.md

```diff
@@ -0,0 +1,73 @@
+# GSI Satbias Converter
+
+GSI Satbias Converter is a tool that converts GSI files with bias correction coefficients (satbias)
+to the NetCDF files written using IODA ObsGroup capability. One NetCDF file is created for each
+sensor.
+
+## Configuration
+
+The configuration file specifies which sensors need to be processed from the input GSI satbias files,
+the output file names and the names of the predictors for the new file.
+An example of a configuration file can be found in `ioda_converters/test/testinput/satbias_converter.yaml`:
```

Comment:
can you turn this into a hyperlink:
[`test/testinput/satbias_converter.yaml`](https://github.com/JCSDA-internal/ioda-converters/blob/develop/test/testinput/satbias_converter.yaml)

Reply:
Done, thank you!